src/core/sys/posix/signal.d

```diff
@@ -1181,6 +1222,54 @@ else version( OpenBSD )
     int sigsuspend(in sigset_t *);
     int sigwait(in sigset_t*, int*);
 }
+else version( DragonFlyBSD )
+{
+    enum SIG_HOLD = cast(sigfn_t2) 3;
```

Comment:
`enum sigfn_t2`

Reply:
@wilzbach Can you clarify what you want me to do ?

Reply:
He's saying to just set the type of the enum, rather than casting the value.  Does that work for a function pointer like this?

Reply:
Done... Thanks !
Note: if this is an important change, the other version(OS) entries do perform the same cast, so you might want to make a similar change to the other implementations (in a separate PR).